LEGO1/mxbitmap.cpp

```diff
@@ -10,23 +10,22 @@ MxBitmap::MxBitmap()
   this->m_bmiHeader = NULL;
   this->m_paletteData = NULL;
   this->m_data = NULL;
-  this->m_unk18 = 0;
+  this->m_unk18 = MX_FALSE;
   this->m_palette = NULL;
 }
 
 // OFFSET: LEGO1 0x100bca10
 MxBitmap::~MxBitmap()
 {
   if (this->m_info != NULL) {
-    free(this->m_info);
+    delete m_info;
```

Comment:
It's probably better to be consistent in the use of explicit `this->`, i.e. having `if (this->m_info)` followed by `delete m_info;` is inconsistent. (applies to the whole file).

Reply:
I'm not sure, does it cause a difference in the matching? And most likely delete is what is actually in the code

Reply:
`this->m_foo` is identical to plain `m_foo` for both parsing and code-generation purposes.

To clarify, my comment was referring to (simplified for brevity):
inconsistent1: `if (this->m_foo) delete m_foo;`
inconsistent2: `if (m_foo) delete this->m_foo;`
consistent1: `if (this->m_foo) delete this->m_foo;`
consistent2: `if (m_foo) delete m_foo;`

The only places `this->` is required is _some_ template code, which is way deeper down a rabbit hole than we shall go here. :) Sometimes it can be of documentation benefit to keep `this->` prefix in rev. code, but whatever is selected it should be applied consistently.

I can also straighten out any question marks regarding use of `delete` in the actual code. It's not "most likely", it's definitely. 100%. 0x10086260 is `??3@YAXPAX@Z`, a.k.a. the global operator delete function.

Reply:
iirc for the MxTimer matching, I think sometimes the MxTimer::member variables made a difference, but idk, I want to move onto MxPalette once this is merged